python/masonite/config/middleware.py

```diff
@@ -0,0 +1,33 @@
+"""Middleware Configuration Settings."""
+
+from masonite.middleware import ResponseMiddleware, MaintenanceModeMiddleware
+
+from app.http.middleware.AuthenticationMiddleware import \
+    AuthenticationMiddleware
+from app.http.middleware.CsrfMiddleware import CsrfMiddleware
+from app.http.middleware.LoadUserMiddleware import LoadUserMiddleware
+from app.http.middleware.VerifyEmailMiddleware import VerifyEmailMiddleware
+
+"""HTTP Middleware
+HTTP middleware is middleware that will be ran on every request. Middleware
+is only ran when a HTTP call is successful (a 200 response). This list
+should contain a simple aggregate of middleware classes.
+"""
+
+HTTP_MIDDLEWARE = [
```

Comment:
I can remove or comment out unused middleware

Reply:
@josephmancuso do you have any suggestion making this setup leaner?
- I have commented out the middleware. 
- I have commented out third-party providers

Following are simple controller method which will be tested against the benchmarks. 

https://github.com/the-benchmarker/web-frameworks/pull/1249/files#diff-5f3a0e2d514d4ef3d3c2e62feb030f9eR5